index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+                      <dfn>Physical Minimum tag</dfn>, <dfn>Physical Maximum
+                      tag</dfn>, <dfn>Unit Exponent tag</dfn>, <dfn>Unit
+                      tag</dfn>, <dfn>Report Size tag</dfn>, <dfn>Report ID
+                      tag</dfn>, or <dfn>Report Count tag</dfn> modifies the
+                      respective field of the global item state table. An item
+                      with the <dfn>Push tag</dfn> pushes a copy of the current
+                      global item state table onto the item state stack. An
+                      item with the <dfn>Pop tag</dfn> replaces the global item
+                      state table with the top of the item state stack. The
+                      global Report ID value is not affected when manipulating
+                      the item state stack.
+                    </p>
+                  </div>
+                </li>
+                <li>If |item| is a [=Global item=]:
+                  <ol>
+                    <li>Let |globalState| be the last item of |globalStack|.
+                    </li>
+                    <li>Let |value:unsigned long| be the [=item data field=]
+                    interpreted as an `unsigned long`.
+                    </li>
+                    <li>If |item| has the [=Usage Page tag=], then set
+                    |globalState|[`"usagePage"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Logical Minimum tag=], then set
+                    |globalState|[`"logicalMinimum"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Logical Maximum tag=], then set
+                    |globalState|[`"logicalMaximum"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Physical Minimum tag=], then set
+                    |globalState|[`"physicalMinimum"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Physical Maximum tag=], then set
+                    |globalState|[`"physicalMaximum"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Report Size tag=], then set
+                    |globalState|[`"reportSize"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Report ID tag=], then set
+                    |globalState|[`"reportId"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Report Count tag=], then set
+                    |globalState|[`"reportCount"`] to |value|.
+                    </li>
+                    <li>If |item| has the [=Unit tag=]:
+                      <ol>
+                        <li>Let |nibbles| be the result of interpreting |value|
+                        as a sequence of eight 4-bit signed two's complement
+                        integers starting from the low-order bits.
+                        </li>
+                        <li>If |nibbles|[0] is:<br>
```

Comment:
I think it would be cleaner to use an &lt;ul> tag here as well.

Reply:
Done